docker/Dockerfile.watch_api_renewer

```diff
@@ -0,0 +1,17 @@
+FROM public.ecr.aws/lambda/python:3.13
```

Comment:
would be nice to co-locate these dockerfiles with their code, so we could just use `Dockerfile` in each directory, not suffix them with stuff like `.watch_api_renewer`